omod/src/main/java/org/openmrs/module/smartonfhir/web/filter/CORSFilter.java

```diff
@@ -29,12 +29,16 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 	        throws ServletException, IOException {
 		// CORS "pre-flight" request
 		response.addHeader("Access-Control-Allow-Credentials", "true");
-		response.addHeader("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE");
+		response.addHeader("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, OPTIONS");
```

Comment:
Also you can remove the all the methods from here leaving the `OPTIONS` method. I think that's what @ibacher  suggested.

Reply:
@VaishSiddharth made the change PTAL